topics/galaxy-ui/tutorials/rstudio/tutorial.md

```diff
@@ -346,6 +346,10 @@ Here is one last bonus we will mention about RStudio. It's difficult to remember
 >
 {: .hands_on}
 
+## Stopping RStudio
+
+If you want to stop your RStudio interactive tool, once you finish your R treatment and if relevant, saving your RStudio environment in your Galay history, you just have to delete the corresponding history dataset named `RStudio` and showing a in progress state, so yellow.
```

Comment:
You could also mention the overview page at: `User → Active InteractiveTools.` 

And maybe explicitly that data is lost that is not trasferred to Galaxy into a persistent storage.

Reply:
oh yeah! eagle eye !

Reply:
Could you put the explanation as a snippet (then we can call it from the 4 R tutorials)? Thanks a lot

Reply:
True!

Reply:
ping